main/src/ca/uwaterloo/flix/api/lsp/SymbolInformation.scala

```diff
@@ -0,0 +1,33 @@
+package ca.uwaterloo.flix.api.lsp
+
+import org.json4s.JsonDSL._
+import org.json4s._
+
+/**
+  * Represents a `SymbolInformation` in LSP.
+  *
+  * @param name           The name of this symbol
+  * @param kind           The kind of this symbol
+  * @param location       The location of this symbol. The location's range is used by a tool
+  *                       to reveal the location in the editor. If the symbol is selected in the
+  *                       tool the range's start information is used to position the cursor. So
+  *                       the range usually spans more then the actual symbol's name and does
+  *                       normally include things like visibility modifiers.
+  *                       The range doesn't have to denote a node range in the sense of a abstract
+  *                       syntax tree. It can therefore not be used to re-construct a hierarchy of
+  *                       the symbols.
+  * @param containerName  The name of the symbol containing this symbol. This information is for
+  *                       user interface purposes (e.g. to render a qualifier in the user interface
+  *                       if necessary). It can't be used to re-infer a hierarchy for the document
+  *                       symbols.
+  * @param tags           Tags for this symbol.
+  */
+case class SymbolInformation(name: String, kind: SymbolKind, location: Location, containerName: Option[String], tags: List[SymbolTag] = List()) {
```

Comment:
I would request that this is removed :)

Our codebase is simply to big to have unused- may be useful in the future- code :)

Reply:
Sure no problem!